src/Resources/translations/admin.nl.yml

```
@@ -1,3 +1,15 @@
+admin:
+    duplicate:
+        title_format: '[KOPIE] - %title%'
+    help:
+        override: Klik op bovenstaande button om de originele pagina <strong>%copied_from%</strong> te overschrijven met de huidige pagina
```

```suggestion
        override: Klik op bovenstaande button om de originele entiteit <strong>%copied_from%</strong> te overschrijven met de huidige entiteit
```
Het kan vanalles zijn...